### PR TITLE
Wrong draw call name for vkCmdBindDescriptorSets

### PR DESCRIPTION
**Pull request by Ryp**

### CODE REVIEW

**Comment by ziga-lunarg** [comment]
I believe `vkUpdateDescriptorSet*` is used to include both `vkUpdateDescriptorSets` and `vkUpdateDescriptorSetWithTemplate`

**Comment by ncesario-lunarg** [comment]
@ziga-lunarg yes, pretty sure that is correct. We should either leave this as is, or perhaps reword to something like:
```
... at vkUpdateDescriptorSet, vkUpdateDescriptorSetsWithTemplate, vkCmdPushDescriptorSet, vkCmdPushDescriptorSetWithTemplate  or similar call ...
```
but a specific list of calls has the drawback that that it will get outdated with near certainty (though this would hopefully be clear from the "or similar call" part). Ideally we would have some way of tracking the exact call that is responsible, but this is not currently done.

@Ryp what do you think would be clearer?

**Comment by Ryp** [comment]
I don't mind the * notation, we should at least patch the base command name because it's wrong for now.

File: layers/descriptor_sets.cpp [hunk 1]
```diff
@@ -1225,7 +1225,7 @@ bool CoreChecks::ValidateImageDescriptor(const char *caller, const DrawDispatchV
                     return LogError(set, vuids.descriptor_valid,
                                     "Descriptor set %s encountered the following validation error at %s time: Image layout "
                                     "specified "
-                                    "at vkUpdateDescriptorSet* or vkCmdPushDescriptorSet* time "
+                                    "at vkUpdateDescriptorSet* or vkCmdBindDescriptorSet* time "
```
Comment (ncesario-lunarg) [inline layers/descriptor_sets.cpp:1228]: `vkCmdPushDescriptorSet*` can still trigger this call, no? Is there a reason it is removed?

Comment (spencer-lunarg) [reply layers/descriptor_sets.cpp:1228]:
```suggestion
                                    "at vkCmdBindDescriptorSets time "
```

Looking at the VUID, It specifically says 

> Descriptors in each bound descriptor set, specified via vkCmdBindDescriptorSets

@Ryp I realize this has been sitting around a while (sorry), would you mind rebasing with the fix, if you would prefer me to apply the change instead I would also be willing to do that

**Pull request closed without merging**